src/hotspot/share/oops/instanceKlass.cpp

```diff
@@ -4208,6 +4208,11 @@ void InstanceKlass::log_to_classlist(const ClassFileStream* stream) const {
     bool skip = false;
     if (is_shared()) {
       assert(stream == NULL, "shared class with stream");
+      if (is_hidden()) {
+        // Not including archived lambda proxy class in the classlist.
```

Comment:
I think it's clearer to say `// Don't include archived lambda proxy class in the classlist.`